services/static-webserver/client/source/class/osparc/auth/LoginPageS4L.js

```diff
@@ -52,12 +52,8 @@ qx.Class.define("osparc.auth.LoginPageS4L", {
 
     __setBackgroundImage: function() {
       let backgroundImage = "";
-      const today = new Date();
-      const xmas = new Date(2023, 12, 24);
-      const endDay = new Date(new Date().setDate(xmas.getDate() + 12));
-      const isXmasDays = today < endDay;
 
-      const defaultBG = isXmasDays ? "url(resource/osparc/Santa_Billie.png)" : "url(resource/osparc/Sim4Life_login_page_master_transparent_bg.png)";
+      const defaultBG = "url(resource/osparc/Sim4Life_login_page_master_transparent_bg.png)";
```

Comment:
Q: sorry, here a recurrent question: is there a way to put these assets in https://github.com/ZurichMedTech/s4l-assets instead?

Reply:
this image is being removed. It's not the best idea to have internal product assets outside your repo as some one might delete not knowing there is a dependency on it. I can add the image there as a back up?